pywbem/cim_operations.py

```diff
@@ -1131,6 +1180,22 @@ def last_reply_len(self):
         """
         return self._last_reply_len
 
+    @property
+    def conn_id(self):
+        """
+        :term:`unicode string`:
```

Comment:
Introduce a term for it, e.g. "connection id".

Reply:
Created a new term named connection id. DONE